apps/crn-server/src/data-providers/contentful/news.data-provider.ts

```diff
@@ -76,6 +77,6 @@ export const parseContentfulGraphQlNews = (item: NewsItem): NewsDataObject => ({
   thumbnail: item.thumbnail?.url ?? undefined,
   link: item.link ?? undefined,
   linkText: item.linkText ?? undefined,
-  text: item.text ? parseRichText(item?.text.json) : undefined,
+  text: item.text ? parseRichText(item?.text as RichTextFromQuery) : undefined,
```

Comment:
this should work
```suggestion
  text: item.text ? parseRichText(item.text as RichTextFromQuery) : undefined,
```
also why do we have to assert?

Reply:
this `parseRichText` will receive the text field from pages and news. I didn't want to make the type to be the one autogenerated from news or pages because I think it would be weird to choose one of them. I didn't discover a smart way to do that, so I wrote the type manually